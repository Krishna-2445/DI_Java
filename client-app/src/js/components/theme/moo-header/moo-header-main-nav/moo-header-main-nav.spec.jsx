import React from 'react';
import { shallow } from 'enzyme';
import { MenuItem } from 'semantic-ui-react';
import MooHeaderMainNav from './moo-header-main-nav';

describe('<MooHeaderMainNav />', () => {
  it('renders correct text and links', () => {
    const links = [
      { text: 'Home', to: '/home' },
      { text: 'Away', to: '/away' },
    ];
    const wrapper = shallow(<MooHeaderMainNav links={links} />);
    expect(wrapper.contains('Home')).toEqual(true);
    expect(wrapper.contains('Away')).toEqual(true);
    expect(wrapper.find(MenuItem)).toHaveLength(2);
    expect(wrapper).toMatchSnapshot();
  });
});
