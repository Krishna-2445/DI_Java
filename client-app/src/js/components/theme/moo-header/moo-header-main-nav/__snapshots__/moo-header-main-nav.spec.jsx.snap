// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MooHeaderMainNav /> renders correct text and links 1`] = `
<Menu
  borderless={true}
  className="moo-menu-bar"
  floated={true}
>
  <MenuItem
    activeClassName="active"
    as={[Function]}
    to="/home"
  >
    Home
  </MenuItem>
  <MenuItem
    activeClassName="active"
    as={[Function]}
    to="/away"
  >
    Away
  </MenuItem>
</Menu>
`;
